action.py

```diff
@@ -1,10 +1,9 @@
-import json
-
 import click
+import json
 import requests
 
-base_url = "http://172.27.46.118:8080/ga4gh/drs/v1/objects"
-base_info_url = "http://172.27.46.118:8080/ga4gh/drs/v1/service-info"
+base_url = "http://172.20.144.25:8080/ga4gh/drs/v1/objects"
```

Comment:
Are these IPs fixed, or will they change every time you restart your computer?

You could also make this part of the CLI - provide a command line option for the server base IP, and provide a default setting, such as 'localhost' or '127.0.0.1', etc. That way a user can run it regardless of where the server is by providing the URL as a parameter.

Reply:
I will do!